spec/models/item_spec.rb

```diff
@@ -199,15 +199,6 @@
   end
 
   context "Methods >" do
-    describe "storage_locations_containing" do
-      it "retrieves all storage locations that contain an item" do
```

Comment:
I guess this is deleteable mostly because it would need to be completely re-written. Ah -- and it is effectively covered by spec/models/view/inventory_spec.rb `#storage_locations_for_item`. OK cool.